Sources/VLCMovieViewController.m

```diff
@@ -1232,41 +1294,63 @@ - (IBAction)sleepTimerAction:(id)sender
     [self setControlsHidden:YES animated:YES];
 }
 
-- (void)moreActions:(UIButton *)sender
-{
+- (void)toggleMultiSelectionView:(UIButton *)sender {
     if (_multiSelectionView.hidden == NO) {
         [UIView animateWithDuration:.3
                          animations:^{
                              self->_multiSelectionView.hidden = YES;
                          }
-                         completion:^(BOOL finished){
-                         }];
+                         completion:nil];
         return;
     }
-
-    if (UI_USER_INTERFACE_IDIOM() == UIUserInterfaceIdiomPhone) {
-        if (UIInterfaceOrientationIsLandscape([[UIApplication sharedApplication] statusBarOrientation])) {
-            _multiSelectionView.showsEqualizer = YES;
-        } else {
-            _multiSelectionView.showsEqualizer = NO;
-        }
-    }
-
+    
+    BOOL isLandscape = UIInterfaceOrientationIsLandscape([[UIApplication sharedApplication] statusBarOrientation]);
+    BOOL isIphone = (UI_USER_INTERFACE_IDIOM() == UIUserInterfaceIdiomPhone);
+    _multiSelectionView.showsEqualizer = isLandscape && isIphone;
+    
```

Comment:
same logic issue

Reply:
I think the old logic was the same: the code was: 
`if (UI_USER_INTERFACE_IDIOM() == UIUserInterfaceIdiomPhone) {
        if (UIInterfaceOrientationIsLandscape([[UIApplication sharedApplication] statusBarOrientation])) {
            _multiSelectionView.showsEqualizer = YES;
        } else {
            _multiSelectionView.showsEqualizer = NO;
        }
}`

I changed it back to match the other one though

Reply:
yes but it wouldn't enter the block for iPad therefor _multiSelectionView.showsEqualizer would be still true

Reply:
the original code is : 
```
if (UI_USER_INTERFACE_IDIOM() != UIUserInterfaceIdiomPhone) {		
         multiSelectionFrame = (CGRect){CGPointMake(0., 0.), [_multiSelectionView proposedDisplaySize]};		
         multiSelectionFrame.origin.x = controllerPanelFrame.size.width - multiSelectionFrame.size.width;		
         multiSelectionFrame.origin.y = controllerPanelFrame.origin.y - multiSelectionFrame.size.height;		
         _multiSelectionView.frame = multiSelectionFrame;		
         _multiSelectionView.showsEqualizer = YES;		
         return;		
     }
```